locales/en-US/app.l20n

```diff
@@ -50,7 +50,7 @@
 
 <notFoundHeader "Four Oh Four!">
 
-<experimentListPageHeader "Now testing!">
+<experimentListPageHeader "Ready for takeoff!">
```

Comment:
John's comment in #969 has the 't' in takeoff capitalized, good to merge once this is updated